.gitignore

```diff
@@ -16,3 +16,7 @@
 
 # IDEs
 .vscode/
+
+# Prevent configuration file from being uploaded
+*.yaml
```

Comment:
Kind of dangerous. Many tools use a yaml or json file on the project's root for configuration. Take a look at cibyl as an example.

Reply:
Fixed, I will only add specific config file for syncron